runtime/compiler/x/codegen/J9TreeEvaluator.cpp

```diff
@@ -12081,7 +12580,24 @@ J9::X86::TreeEvaluator::directCallEvaluator(TR::Node *node, TR::CodeGenerator *c
          else
             break;
          }
-
+      case TR::java_lang_StringCoding_countPositives:
+         {
+            if (comp->target().cpu.supportsAVX())
+               {
+               return inlineCountPositives(node, cg);
+               }
+         }
+         break;
+#if JAVA_SPEC_VERSION < 19
```

Comment:
Excluding this call for JDK < 19 is OK, but why do the same for the method declaration?